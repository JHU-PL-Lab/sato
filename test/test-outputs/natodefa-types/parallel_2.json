[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [],
          "err_match_val": "{}",
          "err_match_expected": "Boolean",
          "err_match_actual": "Record {}"
        }
      ]
    ],
    "err_input_seq": [ 1 ],
    "err_location": "{} or false",
    "err_steps": 333
  },
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [],
          "err_match_val": "fun x ->\nx",
          "err_match_expected": "Integer",
          "err_match_actual": "Function"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "(fun x ->\nx) + 1",
    "err_steps": 333
  }
]