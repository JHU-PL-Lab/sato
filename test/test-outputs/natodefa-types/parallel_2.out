Type errors for:
- Input sequence : [0]
- Found at expr  : (fun x ->
x) + 1
--------------------
* Value    : fun x -> x
* Expected : Integer
* Actual   : Function
Type errors for:
- Input sequence : [1]
- Found at expr  : {} or false
--------------------
* Value    : {}
* Expected : Boolean
* Actual   : Record {}
2 errors found. No further control flows exist.
